src/decaylanguage/modeling/__init__.py

```diff
@@ -2,4 +2,4 @@
 from .amplitudechain import LS
 from .amplitudechain import AmplitudeChain
 
-__all__ = (LS, AmplitudeChain)
+__all__ = ("LS", "AmplitudeChain")
```

Comment:
These should always be strings.

Reply:
Your baby, did not touch that :D.

Reply:
I'm likely the author of many/most of the issues, just pointing things out for instructional purposes in the future for all of us. :)

Reply:
Oh, most probably me writing non-optimal code overall. I bet that ;-). But yes, your comments are very much appreciated for exactly the reason you say. Thank you for that.